### ISSUE TITLE
Feature Request:  Show NEW or UPDATED article filters

### ISSUE THREAD

**Issue by BendixSpring** [TeamMentor/TM_4_0_Design#345]
It would be great to have a button that would show new or updated articles.  Maybe even allowing the user to specify how recent...

**Comment by BendixSpring** [comment]
Maybe less is more.  The current home page is pretty busy with 250 articles.

Maybe the default should be 10 new or updated articles....

**Comment by DinisCruz** [comment]
yeah I agree, see https://github.com/TeamMentor/TM_4_0_Design/issues/298 

I think we shouldn't show more than 20 to 50 article at once

We need to find a good balance here on how much info to show

**Comment by BendixSpring** [comment]
I’m thinking like 5 or 6 articles.  Less is more.

**Comment by roman87** [comment]
Yep, this is exactly what is planned for here: https://github.com/TeamMentor/Master/issues/945

**Comment by romichg** [comment]
This needs to rely on the EDITOR interface. As currently the editing of the articles is de-coupled in such a way that it is difficult to get the update time/data.

**Comment by BendixSpring** [comment]
I think knowing how fresh information is can be quite helpful.  Particularly for a regular user, knowing what's new or updated is very valuable.  I hope we don't lose this data point by closing this issue.

**Comment by romichg** [comment]
I absolutely agree! I didn't close this one :) I just moved it to the BACKLOG so it is better organized and can be picked up in the future sprint.

**Comment by romichg** [comment]
Related to #338
